manifests/s/Spotify/Spotify/1.1.77.643.g3c4c6fc6/Spotify.Spotify.installer.yaml

```
@@ -18,6 +18,6 @@ ElevationRequirement: elevationProhibited
 Installers:
 - Architecture: x86
   InstallerUrl: https://download.scdn.co/SpotifyFullSetup.exe
-  InstallerSha256: 875879C06A8C9096DB2D7AD56C60703F54C5ED39CC61730A72F7096CDBC89720
+  InstallerSha256: 4e47f6fb28d328da5302a1b10a1758e03d9dc743838e5ef0d25f60b2a3983721
```

Are you sure that is the SHA256?

```
1/20/2022 6:10:49 PM
Status: Completed
##[error] Installer https://download.scdn.co/SpotifyFullSetup.exe
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   BB586A8125AFC8170C8CDA4A6A16A6FB79E174086EAD4730D7FB94FFE461DF24
Expected hash: 4e47f6fb28d328da5302a1b10a1758e03d9dc743838e5ef0d25f60b2a3983721
```
```

PS M:\> Get-FileHash -Path C:\Users\User\Downloads\SpotifyFullSetup.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          BB586A8125AFC8170C8CDA4A6A16A6FB79E174086EAD4730D7FB94FFE461DF24       C:\Users\User\Downloads\Spoti...
```

```suggestion
  InstallerSha256: BB586A8125AFC8170C8CDA4A6A16A6FB79E174086EAD4730D7FB94FFE461DF24
```

---

yes, sorry, was the hash for the SpotifySetup.exe and not for the SpotifyFullSetup.exe -> corrected